### ISSUE TITLE
Extension stopped working, getting text suggestions instead

### ISSUE THREAD

**Issue by plaa** [NascHQ/nasc-vscode-mac-touchbar#36]
The extension recently just stopped working, and I'm getting text suggestions instead:

<img width="1085" alt="Touch Bar Shot 2019-11-12 at 7 35 37" src="https://user-images.githubusercontent.com/1436281/68645084-9d55ea80-051f-11ea-91d9-ea5b002d233c.png">

I'm guessing this started with an upgrade of VScode.  Currently running VScode 1.40.0 and Nasc 1.12.0.

Any idea what could be wrong and how to get it working again?

**Comment by wvxavier** [comment]
Same here

**Comment by plaa** [comment]
Nope. I only have two buttons defined.  I noticed that if I don't have any editors open, the nasc buttons show. But with no files open, it's not much use...

**Comment by bessoltsev** [comment]
Work for me! Turn off in global settings of touch bar "Show text suggestions". For a temporary solution will go)

**Comment by plaa** [comment]
@bessoltsev Thanks, disabling text suggestions from global Mac preferences makes the Nasc buttons show. I didn't use the text suggestions anyway, so no problem for me. But in general this may be a problem for many(?) users.

**Comment by bessoltsev** [comment]
Agree! Sometimes it seems that this extension is main reason to use touch bar)

**Comment by pdfowler** [comment]
Depending on how many actions you have enabled in your config, you may be able to collapse the Text Suggestion using the right-facing chevron icon on the left side of the three Touch Bar suggestions. This will collapse the text suggestions and show the nasc Touch Bar actions.

However, if you have too many actions enabled, they may not be shown

**Comment by gregwym** [comment]
Thanks to @pdfowler , I chose to hide the "run without debug" button `workbench.action.debug.run` to free up some room.
